apps/console-electron/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@dicedb/console-electron",
+  "version": "0.1.0",
+  "private": true,
+  "main": "main.js",
+  "type": "module",
+  "scripts": {
+    "dev": "concurrently --kill-others \"next dev -p ${PORT-4000}\" \"electron .\"",
+    "dev:next": "next dev",
+    "build": "next build",
```

Comment:
Seems like we are missing the electron builds and packaging steps

Reply:
That is a different beast, I am thinking of tackling that in a separate PR.

Reply:
Makes sense, lets do that